x-pack/elastic-agent/pkg/agent/cmd/common.go

```diff
@@ -75,6 +75,10 @@ func NewCommandWithArgs(args []string, streams *cli.IOStreams) *cobra.Command {
 	cmd.AddCommand(newEnrollCommandWithArgs(flags, args, streams))
 	cmd.AddCommand(newInspectCommandWithArgs(flags, args, streams))
 
+	// TODO: remove before merging
```

Comment:
this is just to make testing simpler. if you dont want to setup action and fleet, you can invoke this using elastic-agent upgrade. this will be removed when i merge the PR so only way of invoking upgrade will be using fleet